engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/RegexCompileNode.java

```diff
@@ -66,6 +66,8 @@ Object compile(String pattern, String options) {
     try {
       var regex = ctx.parseInternal(src).call();
       return regex;
+    } catch (IllegalStateException ex) {
```

Comment:
When internal `regex` language isn't installed, we get a crash of the engine. Rather than crashing convert the situation into normal Enso panic.